spec/spec_tests/command_monitoring_spec.rb

```diff
@@ -41,7 +41,13 @@ def ignore?(test)
 
         test.expectations.each do |expectation|
 
-          it "generates a #{expectation.event_name} for #{expectation.command_name}", unless: ignore?(test) do
+          before do
```

Comment:
What's the advantage of having the test still generated if we don't want it to run?

Reply:
This way an engineer running the tests knows there are tests which weren't run.